python/amici/setuptools.py

```diff
@@ -203,23 +203,31 @@ def generateSwigInterfaceFiles():
     """Compile the swig python interface to amici
     """
     swig_outdir = '%s/amici' % os.path.abspath(os.getcwd())
-    swig_cmd = find_swig()
-    sp = subprocess.run([swig_cmd,
-                         '-c++',
-                         '-python',
-                         '-Iamici/swig', '-Iamici/include',
-                         '-DAMICI_SWIG_WITHOUT_HDF5',
-                         '-outdir', swig_outdir,
-                         '-o', 'amici/amici_wrap_without_hdf5.cxx',
-                         'amici/swig/amici.i'])
+    swig_exe = find_swig()
+
+    # Swig AMICI interface without HDF5 dependency
+    swig_cmd = [swig_exe,
+                '-c++',
+                '-python',
+                '-Iamici/swig', '-Iamici/include',
+                '-DAMICI_SWIG_WITHOUT_HDF5',
+                '-outdir', swig_outdir,
+                '-o', 'amici/amici_wrap_without_hdf5.cxx',
+                'amici/swig/amici.i']
+    print(f"Running SWIG: {' '.join(swig_cmd)}")
+    sp = subprocess.run(swig_cmd)
     assert (sp.returncode == 0)
     shutil.move(os.path.join(swig_outdir, 'amici.py'),
                 os.path.join(swig_outdir, 'amici_without_hdf5.py'))
-    sp = subprocess.run([swig_cmd,
-                         '-c++',
-                         '-python',
-                         '-Iamici/swig', '-Iamici/include',
-                         '-outdir', swig_outdir,
-                         '-o', 'amici/amici_wrap.cxx',
-                         'amici/swig/amici.i'])
+
+    # Swig AMICI interface with HDF5 dependency
+    swig_cmd = [swig_exe,
+                '-c++',
+                '-python',
+                '-Iamici/swig', '-Iamici/include',
+                '-outdir', swig_outdir,
+                '-o', 'amici/amici_wrap.cxx',
+                'amici/swig/amici.i']
+    print(f"Running SWIG: {' '.join(swig_cmd)}")
```

Comment:
can we explicitetly state with/without HDF5 in the printed messages. Might avoid some confusion why swig is run twice.

Reply:
it's part of the output:
```
    Running SWIG: 'swig3.0 -c++ -python -Iamici/swig -Iamici/include -DAMICI_SWIG_WITHOUT_HDF5 -outdir /home/dweindl/src/AMICI-devel/python/sdist/amici -o amici/amici_wrap_without_hdf5.cxx amici/swig/amici.i'
    Running SWIG: 'swig3.0 -c++ -python -Iamici/swig -Iamici/include -outdir /home/dweindl/src/AMICI-devel/python/sdist/amici -o amici/amici_wrap.cxx amici/swig/amici.i'
``` 
Should be clear that it's doing different things, not?

Reply:
true, should be okay!